index.php

```diff
@@ -20,10 +20,46 @@
 if (isset($_GET['breed'])) {
     $cats = $catHydrator->createCatEntitiesArray((int)$_GET['breed']);
     foreach ($cats as $cat) {
-        $catsHtml .= '<div class="cat-image"><img src="' . $cat->getImage() . '" alt="' . $breeds[$cat->getBreed() - 1]['breed'] . '"></div>';
+        $catsHtml .= '<div class="cat-image">
+        <div class="favorite-icon-container">
+        <form action="index.php" method="post">
+        <input class="cat-id-input" name="breedID" value="' . $cat->getBreed() . '">
+        <input class="cat-id-input" name="newFavourite" value="' . $cat->getID() . '">
+        <img class="favorite-icon" src="';
```

Comment:
naming inconsistency - I don't mind which you use, and I don't mind if the text in the UI doesn't match the naming in the app source code, but you have to pick one. I recommend whichever spelling you're using in the table

Reply:
on it